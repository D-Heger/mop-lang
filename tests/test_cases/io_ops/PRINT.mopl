; Basic print test
PRINT "test"
PUSH 1
PRINT TOP
PRINT "test newline\ntest newline"
PRINT "test hor tab\ttest hor tab"
PRINT "test \\"
PRINT "\""
HALT