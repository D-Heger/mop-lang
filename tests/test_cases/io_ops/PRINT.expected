test
1
test newline
test newline
test hor tab	test hor tab
test \
"